core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -793,7 +793,7 @@ public void heartbeat() {
         synchronized (worker) {
           final long lastUpdate = mClock.millis() - worker.getLastUpdatedTimeMs();
           if (lastUpdate > masterWorkerTimeoutMs) {
-            LOG.error("The worker {} timed out after {}ms without a heartbeat!", worker,
+            LOG.error("The worker {} timed out after {}ms without a heartbeat!", worker.getId(),
```

Comment:
I still think the hostname would be useful for debugging. Isn't the worker id simply a random number?

Reply:
how about both ID and hostname? most of the logging message has only worker Id , having worker Id is easier for us to grep all events with this worker Id. With worker hostname is definitely more useful for us to read this message.

The problem with current logging format is that it has too much information including all blocks Ids on this worker in this single message

Reply:
Yeah, I was not clear, but that was what I was thinking too. I wanted to add the hostname, so the full message would have both id and hostname.

Reply:
fixed